tools/wptrunner/wptrunner/executors/executorwebdriver.py

```diff
@@ -517,7 +517,7 @@ def is_alive(self):
             # 5 seconds of extra_timeout we have as maximum to end the test before
             # the external timeout from testrunner triggers.
             self.webdriver.send_session_command("GET", "window", timeout=2)
-        except (socket.timeout, error.UnknownErrorException, error.InvalidSessionIdException):
+        except (OSError, error.UnknownErrorException, error.InvalidSessionIdException):
```

Comment:
At L635 of this file, it says is_alive is False if got `error.WebDriverException`. Should we move that here?

I kind of think `is_alive` should always return a value, and made https://chromium-review.googlesource.com/c/chromium/src/+/5515941. Maybe we need work with chromedriver team to understand what are the exceptions could be thrown here.

Reply:
> Should we move that here?

Sure (see below)

> Maybe we need work with chromedriver team to understand what are the exceptions could be thrown here.

The spec documents [error codes]. Looks like the main one that's missing here is "no such window"; I would say the protocol is "not alive" in that case.

[error codes]: https://www.w3.org/TR/webdriver/#get-window-handle